packages/autocertifier-client/src/createSelfSignedCertificate.ts

```diff
@@ -0,0 +1,79 @@
+import * as forge from 'node-forge'
+
+interface Certificates {
+    caCert: string
+    caKey: string
+    serverCert: string
+    serverKey: string
+}
+
+export function createSelfSignedCertificate(fqdn: string, validMonths: number, validMilliseconds?: number): Certificates {
```

Comment:
Maybe we don't both `validMonths` and `validMilliseconds` as only one of the parameters is used (and it is not clear which one)